shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/dto/CarminDatasetProcessingDTO.java

```diff
@@ -0,0 +1,95 @@
+package org.shanoir.ng.processing.carmin.dto;
+
+import org.shanoir.ng.processing.carmin.model.ExecutionStatus;
+import org.shanoir.ng.processing.dto.DatasetProcessingDTO;
+
+
+/**
+ * DTO for CarminDatasetProcessing
+ * 
+ * @author KhalilKes
+ */
+public class CarminDatasetProcessingDTO extends DatasetProcessingDTO {
+
+    private String Identifier;
```

Comment:
All attributes shall by Java convention please start with lower case, thank you!

Reply:
Thank you! 
I will do as you suggested below, removing the DTO and use the entity instead.

Reply:
Hi Khalil, I am waiting for your commit then. Thank you!

Reply:
Perfect, thank you!